src/odemis/gui/comp/viewport.py

```diff
@@ -1366,6 +1370,9 @@ def on_drag_start(self, evt):
     def on_drag_end(self, evt):
         """ End the dragging procedure """
         self._dragging = False
+        # Make sure we stop tracking the mouse
+        if evt.EventObject.HasCapture():
+            evt.EventObject.ReleaseMouse()
```

Comment:
I think i tried this in Linux and lost control of the mouse completely somehow - but maybe I did it in the wrong sequence.